### PR TITLE
Update cookiecutter structure

### PR DESCRIPTION
**Pull request by jpichon**
Cookie cutter update

### CODE REVIEW

**Comment by jpichon** [comment]
Ah! Looks like I missed the Travis tests in the cookie cutter update. This will fail because of Python 2 support removal. On it.

File: README.rst
Comment (jguiditta) [inline README.rst]: Is the switch to '.com' required? I know it is available, but does '.org' now work anymore?

**Review by jguiditta — COMMENTED** [review]
Could these commits be separate PRs? I have some thoughts on the structure of the code, but I don't want to hold up the whole patch to discuss, since the basic cookiecutter stuff looks fine, with the exception of the minor question below.

File: README.rst
Comment (jpichon) [reply README.rst]: The cookie cutter update did that, but I can revert that bit back

**Comment by jpichon** [comment]
> Could these commits be separate PRs? I have some thoughts on the structure of the code, but I don't want to hold up the whole patch to discuss, since the basic cookiecutter stuff looks fine, with the exception of the minor question below.

Did it that way (separate commits, one PR) because we can't do dependent PRs in Github. But I can resubmit this with just the cookie cutter update which should be straightforward, and once that's merged submit the rest. Sounds good, thanks for looking at this already!

**Review by jguiditta — APPROVED** [review]
Looks good!

**Comment by jguiditta** [comment]
I guess we don't have mergify yet, I'll merge this now...

**Pull request merged**
